README.md

```diff
@@ -1,21 +1,74 @@
-# oni-demo
-Download a demo version of Open Network Insight, which can be run standalone on a windows laptop using Winpython https://sourceforge.net/projects/winpython/files/latest/download 
+# **Open Network Insight Demo**
 
+ONI-Demo can be downloaded and run standalone on a **Windows Laptop** 
 
-[Instructions for downloading and running the demo] 
+## **Demo SW Requirements**
 
-We use a portable version of python that has all the dependencies and stuff included, you need to download and run this first: 
-http://sourceforge.net/projects/winpython/files/latest/download 
+This demo requires Winpython, which can be downloaded here:
 
-Install it in the root directory as C:\winpython to avoid any awkwardness in file path references. 
+[Winpython Download][1]
+[1]:  https://sourceforge.net/projects/winpython/files/latest/download "Winpython Installer"  
 
-You want to unzip oni-demo-win.zip into the winpython/notebooks folder 
+Note: Install Winpython under root directory as **C:\winpython**    
 
-Run the shortcut "Jupyter Notebook.exe"�  in the WinPython folder.
 
-When you start the notebook server (above), you can access these two links:
+## **Downloading Demo**
 
-http://localhost:8888/files/oni-demo-win/index_sconnects.html
+In order to download Demo code - you can use Download ZIP option from this repository
 
-http://localhost:8888/files/oni-demo-win/storyboard_sconnect.html
+![Alt][2]
 
+[2]: docs/download-zip.bmp "Download ZIP File"
+
+Unzip the file *oni-demo-win.zip* into **C:\winpython\notebooks** folder
+
+## **Running Demo**
+
+In order to run ONI demo, start **Jupyter Notebook.exe** that is located under winpython folder
+
+A Web browser will be opened in the following path
```

Comment:
The contents of the Jupyter window will likely not match those below (they didn't match when I tried)... that's because the jupyter window shows you the contents of your notebooks folder

(some people might think they are missing some stuff if they don't see all the notebooks that you have in this image)